### ISSUE TITLE
Forum badge for core contributors?

### ISSUE THREAD

**Issue by kdmccormick** [openedx/tcril-engineering#393]
These Forum Mod & Tutor Maintainer badges are cool:
![image](https://user-images.githubusercontent.com/3628148/181262865-89da1ba5-1b4d-4583-bb56-eb4960de4492.png)
![image](https://user-images.githubusercontent.com/3628148/181262907-efa64184-ff5b-4644-9a1e-55dfec796493.png)

Would it be hard to do the same thing for Core Contributors, and eventually Maintainers?

**Comment by sarina** [comment]
Nope. I'll note that you can only display one at a time, which is annoying when you hold multiple roles.

**Comment by sarina** [comment]
I'm going to do this for Core Contributors. I'm mixed on "maintainers" because that is a much huger overhead for the few forum mods to have to maintain, and I'm not making 163 badges for each repo. Plus - you can only display one badge at once!

**Comment by kdmccormick** [comment]
Nice!

**Comment by sarina** [comment]
![core contributor badge (1)](https://user-images.githubusercontent.com/1985317/188193817-46d5d818-e740-4c49-a0ef-17423de025b1.png)

**Comment by kdmccormick** [comment]
Cute, I love it.

But will this replace our little company badges?

**Comment by sarina** [comment]
The flair on the profile pic is associated with your email address, and only you can choose to remove it.

The "Tutor Maintainer" flair to the right of your pic is likewise something you can choose. Again - you can only choose 1.

**Comment by sarina** [comment]
See how you have both the coffee cup AND the "tutor maintainer"?

**Comment by kdmccormick** [comment]
Cool, got it.

**Comment by sarina** [comment]
https://discuss.openedx.org/t/new-core-contributor-forums-badge-title/8221
